src/main/java/org/owasp/html/AutoCloseableExtensibleHtmlStreamRenderer.java

```diff
@@ -0,0 +1,96 @@
+package org.owasp.html;
+
+import javax.annotation.WillCloseWhenClosed;
```

Comment:
can you please add original classes links and highlight the new changes

Reply:
added github links in class comment